web/src/components/inputs/select_wide.tsx

```diff
@@ -78,12 +78,8 @@ export function SelectFieldCreatable<T>({ name, label, help, placeholder, toolti
               // value={field?.value ? field.value : options.find(o => o.value == field?.value)}
               value={field?.value ? { value: field.value, label: field.value  } : field.value}
               onChange={(newValue: unknown) => {
-                if (newValue) {
-                  setFieldValue(field.name, (newValue as { value: string }).value);
-                }
-                else {
-                  setFieldValue(field.name, "")
-                }
+                const option = newValue as { value: string };
+                setFieldValue(field.name, option?.value ?? "");
```

Comment:
Won't all of these error if the newValue is undefined for example?

Reply:
I asked myself the same question but i couldn't find a select that triggered that error...